motion/ruby_motion_query/rect.rb

```diff
@@ -368,6 +368,10 @@ def rect_hash_to_rect_array(view, existing_rect, params, grid = nil)
           t = root_view_point.y
         end
 
+        if app.development? && (w <= 0 || h <= 0)
+          mp "#{view.rmq_data.style_name}: One or more values in your frame is a negative or zero. If this is unexpected, check your styles.\nFrame values: {l: #{l}, t: #{t}, w: #{w}, h: #{h}}"
```

Comment:
probably will have to use a `puts` rmq doesnt have a dependency on `motion_print` so we cant depend its there.